### PR TITLE
Cassandra UDT int tests

### PR DESCRIPTION
**Pull request by conorbros**
merge #395 first

### CODE REVIEW

**Comment by rukai** [comment]
Marking as draft until https://github.com/shotover/shotover-proxy/pull/395 is merged as it depends on some of the helpers added there.

File: shotover-proxy/tests/cassandra_int_tests/mod.rs [hunk 1]
```diff
@@ -13,7 +13,6 @@ pub fn cassandra_connection(contact_points: &str, port: u16) -> Session {
     cluster.connect().unwrap()
 }
 
-// first turn ResultValue into derive soup
```
Comment (rukai) [inline shotover-proxy/tests/cassandra_int_tests/mod.rs:16]: LOL

Comment (conorbros) [reply shotover-proxy/tests/cassandra_int_tests/mod.rs:16]: guilty

**Review by rukai — APPROVED** [review]
lgtm

**Review by benbromhead — APPROVED** [review]

**Pull request merged**